docs/source/examples.rst

```diff
@@ -26,4 +26,4 @@ oneAPI Examples
    DPC++ <examples/dpc.rst>
    C++ <examples/cpp.rst>
 
-.. note:: Explore the complete list of oneAPI code samples in the `oneAPI Samples Catalog <https://oneapi-src.github.io/oneAPI-samples/>`_. These samples were designed to help you develop, offload, and optimize multiarchitecture applications targeting CPUs, GPUs, and FPGAs. 
+.. note:: Explore the complete list of oneAPI code samples in the `oneAPI Samples Catalog <https://uxlfoundation.github.io/oneAPI-samples/>`_. These samples were designed to help you develop, offload, and optimize multiarchitecture applications targeting CPUs, GPUs, and FPGAs. 
```

Comment:
same thing https://github.com/uxlfoundation/scikit-learn-intelex/pull/2203#discussion_r1867842380

Reply:
Thanks, reverted the change there.